src/components/Footer/index.js

```diff
@@ -0,0 +1,10 @@
+import Design from "./design";
+
+function Footer() {
+  return (
+    <div>
```

Comment:
why you are using this parent div.?

Reply:
While converting to a 3 file structure I forgot to remove this parent div. I'll fix this.